resolwe_bio/tests/files/56GSID.lf.finalvars.txt

```diff
@@ -1,2 +1,2 @@
 CHROM	POS	ID	REF	ALT	QUAL	DP	AF	SB	DP4	EFF[*].GENE	EFF[*].AA
-2	1640		A	AC	710.0	114	0.236842	4	46;41;11;16	FAM110C	
+2	1640		A	AC	710.0	114	0.236842	4	46;41;11;16	CHR_START-FAM110C	
```

Comment:
`CHR_START` string will probably cause troubles for gene [hyperlinks](https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/tools/make_report.py#L144)?